airflow/migrations/versions/0032_3_0_0_rename_execution_date_to_logical_date_and_nullable.py

```diff
@@ -17,7 +17,7 @@
 # under the License.
 
 """
-Make logical_date nullable.
+Renamed column execution_date to logical_date in both dag_run and log tables. Make logical_date nullable.
```

Comment:
It makes sense to keep the first line simple, short, one line, one sentence.  Can move any other necessary details below.

Reply:
Changed it